tracer/src/Datadog.Trace/AspNet/SharedItems.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="SharedItems.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+
+using System.Collections.Generic;
+using System.Web;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.AspNet
+{
+    internal static class SharedItems
+    {
+        public const string HttpContextPropagatedResourceNameKey = "__Datadog.Trace.ClrProfiler.Managed.AspNetMvcIntegration-aspnet.resourcename";
```

Comment:
I don't really understand why this constant is stored here, when it's never used with the PushItem / PopItem mechanism.

Reply:
It was there and alone in this class in the first place. I decided to use this class as this constant is a key used on the Items dictionary as well.